middleware/auth.js

```diff
@@ -12,44 +12,79 @@ const jwtUtils = require("@utils/jwt-utils")
 // Instantiate router object
 const auth = express.Router()
 
+const { E2E_TEST_REPO, E2E_TEST_SECRET, E2E_TEST_GH_TOKEN } = process.env
+const E2E_TEST_USER = "e2e-test"
+const ALLOWED_URLS = ["/v1/sites", "/v1/auth/whoami"]
+
```

Comment:
Could we also add the v2 endpoints?

Reply:
Renamed the `ALLOWED_URLS` variable to `GENERAL_ACCESS_PATHS` for clarity [f089777](https://github.com/isomerpages/isomercms-backend/pull/270/commits/f0897779566da0f4c585ad5a8a54cfb5eae63384) - this array of paths is supposed to be for paths which all CMS users have access to.